src/application.ts

```diff
@@ -107,7 +111,8 @@ export default class Application implements Owner {
     if (!mainTemplate) { throw new Error("Could not find main template."); }
 
     let mainLayout = templateFactory(mainTemplate).create(this.env);
```

Comment:
Good question. I think @rwjblue has the answer, we have some templates like these in Ember: https://github.com/emberjs/ember.js/tree/master/packages/ember-glimmer/lib/templates